Makefile

```diff
@@ -50,7 +50,7 @@ clean:
 	go clean -cache -testcache ./...
 
 fmt: $(GOFUMPORTS)
-	@$(GOFUMPORTS) -l -w $(shell find internal cmd test -name '*.go')
+	@$(GOFUMPORTS) -l -w $(shell find internal apis controllers test -name '*.go')
```

Comment:
Is `main.go` included here?

Reply:
Nice catch! Will add it to the list. Thanks.

Reply:
We can resolve this comment also @huikang